Tasks/PowerShellV2/powershell.ps1

```diff
@@ -48,7 +48,18 @@ try {
     # Generate the script contents.
     Write-Host (Get-VstsLocString -Key 'GeneratingScript')
     $contents = @()
+
+    # Explicitly set the default output stream preferences, rather then relying on PowerShell's defaults.
+    if ($env:system_debug -eq "true") {
+        $contents += "`$VerbosePreference = 'Continue'"
+    }
+    else {
+        $contents += "`$VerbosePreference = 'SilentlyContinue'"
+    }
+    $contents += "`$InformationPreference = 'Continue'"
```

Comment:
Could it maybe make sense also to make these parameters adjustable via task inputs? This could allow to avoid additional noise by default for some cases, also there could be some possible breaking changes - when output for these streams will contain vso commands which will be interpreted and executed by the agent.